tests/bluetooth/bsim/_compile_permutate_kconfigs.sh

```diff
@@ -62,8 +62,8 @@ perm_compile() {
 	echo "Compile with config overlay:"
 	cat $3
     fi
-    local app=tests/bluetooth/bsim/ll/edtt/hci_test_app
-    local conf_file=prj_dut.conf
+    local app=tests/bluetooth/bsim/edtt_ble_test_app/hci_test_app
+    local conf_file=prj_dut_llcp.conf
```

Comment:
Hmm, still carrying the 'llcp' in the naming - shouldn't we loose that? Perhaps do this in a subsequent PR, as this was introduced to distinguish between legacy and refactored

Reply:
I'll do it in a subsequent PR